app/models/mixins/storage_manager_mixin.rb

```diff
@@ -0,0 +1,28 @@
+module StorageManagerMixin
```

Comment:
@hsong-rh This code is Cinder specific, it shouldn't be in a generic `mixins/storage_manager_mixin.rb` file.